static/js/search.js

```diff
@@ -1280,8 +1280,54 @@ function populateResults(data, full, jumpToSingle) {
           "</div></td></tr>";
       }
 
+      const pathKeys = Object.keys(data[pathkind]);
+      const hasSymbolicIdentity = pathKeys.some(k => 
+        k.startsWith("Definitions") || 
+        k.startsWith("Declarations") || 
```

Comment:
These lines and some more lines below have a trailing whitespace.
Can you remove them?